js/trash.js

```diff
@@ -145,10 +145,15 @@ class Trashcan {
      * @returns {void}
      */
     updateContainerPosition() {
-        const offsetFromBottom = 75;
-        const offsetFromRight = 97;
+        const offsetPercentageFromBottom = 0.1; // 10% from the bottom
```

Comment:
Shouldn't these be based on the size/position of the buttons they are centered over?

Reply:
I think you need to do a rebase... you pulled in lots of unrelated commits into this PR.